### PR TITLE
Added context paths to standard outputs

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Description
Added new context standards outputs to our docs.

### CODE REVIEW

File: docs/context_standards/README.MD [hunk 1]
```diff
@@ -305,6 +310,27 @@ The following is the format for a process.
...
+    "ID": "STRING, The unique ID within the tool retrieving the host.",
+    "DHCPServer": "STRING, DHCP Server.",
+    "IP": "STRING, IP Address of the host.",
+    "MACAddress": "STRING, Host MAC Address.",
+    "Memory": "STRING, Host memory.",
+    "Model": "STRING, Host model.",
+    "OS": "STRING, Host operating system.",
+    "OSVersion": "STRING, Host operating system version.",
+    "Processor": "STRING, Host processor.",
+    "Processors": "STRING, Host processors"
```
Comment (yaakovi) [inline docs/context_standards/README.MD]: why do we have both single and plural of processor?
```suggestion
    "Processors": "STRING, Host processors." 
```

Comment (DeanArbel) [reply docs/context_standards/README.MD]: Because the second field is the Processors number - I'll update the description to better reflect this

**Comment by DeanArbel** [comment]
@kirbles19 Can you please review the (single) file changed?
I'm not sure you had a chance to review this documentation in the page, but if not then now would be a good time to do it, since we're updating our integrations according to this standard, and now we're in the middle of an epic to update all existing integrations.

File: docs/context_standards/README.MD [hunk 2]
```diff
@@ -305,6 +305,26 @@ The following is the format for a process.
...
+## Host
+The following is the format for a host.
+```python
+"Host": {
+    "Domain": "STRING, Domain of the host.",
+    "Hostname": "STRING, Name of the host.",
+    "BIOVersion": "STRING, BIOS version of the host.",
+    "ID": "STRING, The unique ID within the tool retrieving the host.",
+    "DHCPServer": "STRING, DHCP Server.",
+    "IP": "STRING, IP Address of the host.",
```
Comment (kirbles19) [inline docs/context_standards/README.MD]: "IP": "STRING, IP address of the host.",

**Comment by kirbles19** [comment]
@DeanArbel One comment made inline, all done.

File: docs/context_standards/README.MD (hunk 2 above)
Comment (DeanArbel) [reply docs/context_standards/README.MD]: @kirbles19 I'll fix that. Except for that, are the descriptions ok?

**Comment by kirbles19** [comment]
Updated.

**Pull request merged**